.github/workflows/test.yml

```diff
@@ -107,5 +107,15 @@ jobs:
         # because we have a yarn.lock and caches efficiently.
         uses: bahmutov/npm-install@v1
 
+      - name: 📥 Install Playwright
+        run: npx playwright install --with-deps
```

Comment:
Shouldn't we add this as a dependency to the repo, so we can run integration tests locally? 🤔

Reply:
not according to their docs, this way we get the proper browser for linux https://playwright.dev/docs/ci#github-actions